lib/src/test/kotlin/starknet/account/StandardAccountTest.kt

```diff
@@ -19,6 +19,7 @@ import org.junit.jupiter.params.ParameterizedTest
 import org.junit.jupiter.params.provider.MethodSource
```

Comment:
Maybe it'd be also useful to add some kind of a test verifying that trying to execute transaction signed for fee estimation fails?